lib/preflight/getCombinedConnectionStats.ts

```diff
@@ -0,0 +1,141 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { RTCIceCandidateStats, SelectedIceCandidatePairStats,  } from '../../tsdef/PreflightTypes';
+import type { RTCIceCandidatePairStats } from './rtctypes';
+
+interface RTCStatsReport {
+  [x: string]: any;
+  forEach(callbackfn: (value: any, key: string, parent: RTCStatsReport) => void, thisArg?: any): void;
```

Comment:
This is basically `any` type. If we're not declaring all properties, how about doing `type RTCStatsReport = any` instead?

Reply:
somehow with with the declaration as you suggested, I see an error:
Parameter 'stat' implicitly has an 'any' type. when I use forEach later...with the forEach explicitely declared I do not see the same error

Reply:
Ok